docs/sdk/rust/quickstart-rust.mdx

```diff
@@ -0,0 +1,11 @@
+# Rust Quickstart
+
+Rust SDK for Permit.io is coming soon!
+We are planning to release the Rust SDK in the coming weeks (around January 2024) and we'll update this page with the latest information.
```

Comment:
Maybe "this page will be updated with the latest information" ?

Reply:
agree